README.md

```diff
@@ -34,37 +34,61 @@
 
 ## WORK IN PROGRESS
 
-Please note that this plugin is under active development, and might not be ready to run on production code yet.
-
-## Contributing
-
-If you're interested in contributing to the development of Prettier for PHP, you can follow the [CONTRIBUTING guide from Prettier](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md), as it all applies to this repository too.
-
-To test it out on a PHP file:
-
-* Clone this repository.
-* Run `yarn`.
-* Create a file called `test.php`.
-* Run `yarn prettier test.php` to check the output.
+Please note that this plugin is currently in alpha stage and still under active development. We encourage everyone to try it and give feedback, but we don't recommend it for production use yet.
 
 ## Install
 
 yarn:
+
 ```bash
-yarn add --dev prettier/prettier prettier/plugin-php
+yarn add --dev prettier @prettier/plugin-php
+# or globally
+yarn global add prettier @prettier/plugin-php
 ```
 
 npm:
+
 ```bash
-npm install --save-dev prettier/prettier prettier/plugin-php
+npm install --save-dev prettier @prettier/plugin-php
+# or globally
+npm install --global prettier @prettier/plugin-php
 ```
 
 ## Use
 
+If you installed prettier as a local dependency, you can add prettier as a script in your `package.json`,
+
+```json
+scripts: {
+  "prettier":"prettier"
```

Comment:
Do we need glob (`{**/*,*}.php`) here?

Reply:
Not when you're calling it like instructed below: `yarn run prettier path/to/file.php --write`